portfolio.html

```diff
@@ -82,5 +82,237 @@ <h2>I'm Amrendra Glad to see you!</h2>
+          <div class="technology">css</div>
+          <div class="technology">javaScript</div>
+        </div>
+        <button type="button" class="btnseeproject">See Project</button>
+      </div>
+    </section>
+    <!-- ////////// Abmout me Section-->
+
+    <section class="about-myselfsection">
+      <div class="about-myself">About Myself</div>
+      <div>
+        <div class="myself-content">
+          <p>
+            I’m a software developer! skilled in React JS, Express JS,
+            javaScript, CSS, HTML, JAVA, Ruby, Ruby on Rails. Hands-on
+            experience in implementing Web Development technologies. Interested
+            in developing softwares. I can help you build a product , feature or
+            website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+        </div>
+        <div class="aboutconnect-icon">
+          <div class="aboutconnect">LET'S CONNECT</div>
+          <ul class="aboutsocial-icons">
+            <li>
+              <a
+                href="https://twitter.com/amrendrak_"
+                target="_blank"
+                rel="noopener noreferrer"
+                ><i class="fab fa-twitter"></i
+              ></a>
+            </li>
+            <li>
+              <a
+                href="https://www.linkedin.com/in/amrendraakumar/"
+                target="_blank"
+                rel="noopener noreferrer"
+                ><i class="fab fa-linkedin-in"></i
+              ></a>
+            </li>
+            <li>
+              <a
+                href="https://github.com/amrendrakind"
+                target="_blank"
+                rel="noopener noreferrer"
+                ><i class="fab fa-github"></i
+              ></a>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <button type="button" class="btnseeproject">Get my resume</button>
+
+      <!-- //////////// -->
+
+      <div class="acordianwrapper">
+        <button class="accordion">Languages</button>
+        <div class="panel">
+          <div class="languagename">
+            <img src="/Ellipse_JS.png" alt="JSIcon" />
+            JavaScript
+          </div>
+          <div class="languagename">
+            <img src="./Ellipse_HTML.png" alt="HTMLIcon" />
+            HTML
+          </div>
+          <div class="languagename">
+            <img src="./Ellipse_CSS.png" alt="CSSIcon" />
+            CSS
+          </div>
+        </div>
+
+        <button class="accordion">Frameworks</button>
+        <div class="panel">
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+            ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+            aliquip ex ea commodo consequat.
+          </p>
+        </div>
+
```

Comment:
- [ ] Please, consider that the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`
- [ ] Also, the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`